src/crypto_generichash_wrap.cc

```diff
@@ -41,7 +41,7 @@ v8::Local<v8::Value> CryptoGenericHashWrap::NewInstance (unsigned char *key, uns
   v8::Local<v8::Object> instance;
 
   v8::Local<v8::FunctionTemplate> constructorHandle = Nan::New<v8::FunctionTemplate>(crypto_generichash_constructor);
-  instance = Nan::NewInstance(constructorHandle->GetFunction()).ToLocalChecked();
+  instance = Nan::NewInstance(constructorHandle->GetFunction(v8::Isolate::GetCurrent()->GetCurrentContext()).ToLocalChecked()).ToLocalChecked();
```

Comment:
How about this? `instance = Nan::NewInstance(Nan::GetFunction(constructorHandle).ToLocalChecked()).ToLocalChecked();`

Just found it in the Nan test to see how they do it now https://github.com/nodejs/nan/blob/e222068f9d99967897a8cefa6c505534b36116ad/test/cpp/namedinterceptors.cpp#L58-L66